explore/src/main/scala/explore/ExplorePWA.scala

```diff
@@ -75,88 +82,83 @@ object ExplorePWA {
   }
 
   // Check every 10 min maybe too often but we are doing lots of development
-  val intervalMS = 10 * 60 * 1000
-
-  def scheduleUpdateCheck(u: String, r: ServiceWorkerRegistration): Callback = {
-    import scala.concurrent.ExecutionContext.Implicits.global
-    (Callback.log("Check for new explore version") *>
-      pingSW(u)
-        .flatMap(go => Callback(r.update()).when(go).asAsyncCallback)
-        .toCallback
-        .attempt
-        .void)
-      .setIntervalMs(intervalMS.toDouble)
-      .void
+  val updateInteval = FiniteDuration(10, TimeUnit.MINUTES)
+
+  def scheduleUpdateCheck(u: String, r: ServiceWorkerRegistration): IO[Unit] =
+    Stream
+      .awakeEvery[IO](updateInteval)
+      .evalTap { _ =>
+        IO.println("Check for new explore version") >>
+          pingSW(u).flatMap(go => if (go) IO.blocking(r.update()) else IO.unit)
```

Comment:
`blocking` doesn't do anything on JS, because there is no blocking threadpool to move the task to. Anyway, almost nothing is blocking in JS.

```suggestion
          pingSW(u).flatMap(go => if (go) IO(r.update()) else IO.unit)
```

Reply:
```suggestion
          pingSW(u).ifM(IO(r.update()), IO.unit)
```

Reply:
Good point. I clearly wasn't paying attention.